core/domain/question_services_test.py

```diff
@@ -128,12 +132,13 @@ def test_create_and_get_question_skill_link(self):
         # Make sure the correct skill description corresponds to respective
         # question summaries.
         for index, description in enumerate(skill_descriptions):
```

Comment:
Test a case where there are multiple skill descriptions for one question

Reply:
It was tested in line 135-137. If not enough I'll add one more test case.